src/util/ChartUtils.ts

```diff
@@ -1112,7 +1112,7 @@ export const getBandSizeOfAxis = (axis: any, ticks?: Array<TickItem>, isBar?: bo
     return bandSize === Infinity ? 0 : bandSize;
   }
 
-  return 0;
+  return axis?.width ?? 0;
```

Comment:
Hard to say if this fixes an issue in all scenarios. There is a `return` statement two lines above which, also may return 0.